### PR TITLE
Make the powered examine text fully client predicted

### PR DESCRIPTION
**Pull request by Mervill**
## About the PR

The "it appears to be (un)powered." examine text is now client predicted. Meaning all powered objects (a huge section of common objects) should now have much snappier examine text.

## Technical details

Work done by ShadowCommander in #28206 set the stage for this.

## Media

B4
power-examine-fail

After
power-examine-good

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

### CODE REVIEW

File: Content.Shared/Power/EntitySystems/SharedPowerReceiverSystem.cs [hunk 1]
```diff
@@ -1,6 +1,18 @@
...
+
+namespace Content.Shared.Power.EntitySystems;
 
 public abstract class SharedPowerReceiverSystem : EntitySystem
 {
-
+    ///<summary>
+    ///Adds some markup to the examine text of whatever object is using this component to tell you if it's powered or not, even if it doesn't have an icon state to do this for you.
+    ///</summary>
+    protected void OnExamined(EntityUid uid, SharedApcPowerReceiverComponent component, ExaminedEvent args)
```
Comment (Plykiya) [inline Content.Shared/Power/EntitySystems/SharedPowerReceiverSystem.cs:11]: Use `Entity<SharedAPcPowerReceiverComponent>` please

**Review by Plykiya — CHANGES_REQUESTED** [review]

File: Content.Shared/Power/EntitySystems/SharedPowerReceiverSystem.cs (hunk 1 above)
Comment (Mervill) [reply Content.Shared/Power/EntitySystems/SharedPowerReceiverSystem.cs:11]: These are the arguments that are required by `SubscribeLocalEvent`. I don't think there's an overload accepting `Entity<T>`, is there?

Comment (metalgearsloth) [reply Content.Shared/Power/EntitySystems/SharedPowerReceiverSystem.cs:11]: There is.

Comment (deltanedas) [reply Content.Shared/Power/EntitySystems/SharedPowerReceiverSystem.cs]: why did you copy paste it into server and client systems thats what shared is for
`private void OnExamined(Entity<SharedApcPowerReceiverComponent> ent, ref ExaminedEvent args)` in shared and nothing in server or client

**Review by Plykiya — APPROVED** [review]

**Pull request merged**